driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -212,13 +226,29 @@ public InetSocketAddress getSocketAddress() {
    *     href="https://docs.datastax.com/en/cassandra/2.1/cassandra/configuration/configCassandra_yaml_r.html">The
    *     cassandra.yaml configuration file</a>
    */
-  public InetAddress getBroadcastAddress() {
+  public InetSocketAddress getBroadcastSocketAddress() {
     return broadcastAddress;
   }
 
   /**
-   * Returns the node listen address (that is, the IP the node uses to contact other peers in the
-   * cluster), if known.
+   * @return the node listen address, if known. Otherwise {@code null}.
+   * @deprecated
```

Comment:
You might want to explain why it is deprecated ("Use #getListenSocketAddress() instead").